components/hab/src/command/studio/docker.rs

```diff
@@ -77,21 +77,26 @@ pub fn start_docker_studio(_ui: &mut UI, mut args: Vec<OsString>) -> Result<()>
         volumes.push(format!("{}:{}", DOCKER_SOCKET, DOCKER_SOCKET));
     }
 
-    let env_vars = vec![
-        "DEBUG",
-        "HAB_AUTH_TOKEN",
-        "HAB_BLDR_URL",
-        "HAB_BLDR_CHANNEL",
-        "HAB_ORIGIN",
-        "HAB_ORIGIN_KEYS",
-        "HAB_STUDIO_BACKLINE_PKG",
-        "HAB_STUDIO_NOSTUDIORC",
-        "HAB_STUDIO_SUP",
-        "HAB_UPDATE_STRATEGY_FREQUENCY_MS",
-        "http_proxy",
-        "https_proxy",
-        "RUST_LOG",
+    let mut env_vars = vec![
+        String::from("DEBUG"),
+        String::from("HAB_AUTH_TOKEN"),
+        String::from("HAB_BLDR_URL"),
+        String::from("HAB_BLDR_CHANNEL"),
+        String::from("HAB_ORIGIN"),
+        String::from("HAB_ORIGIN_KEYS"),
+        String::from("HAB_STUDIO_BACKLINE_PKG"),
+        String::from("HAB_STUDIO_NOSTUDIORC"),
+        String::from("HAB_STUDIO_SUP"),
+        String::from("HAB_UPDATE_STRATEGY_FREQUENCY_MS"),
+        String::from("http_proxy"),
+        String::from("https_proxy"),
+        String::from("RUST_LOG"),
     ];
+
+    for (key, _) in env::vars().filter(|&(ref key, _)| key.starts_with("HAB_STUDIO_SECRET_")) {
```

Comment:
Think you could make the `HAB_STUDIO_SECRET_` a `const` like `STUDIO_SECRET_ENVVAR_PREFIX`? That would help us to surface these environment variables in the future.